style.scss

```diff
@@ -0,0 +1,130 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins&family=Ubuntu:wght@300&display=swap");
+
+/**
+*! Color variables
+**/
+$color-1: #6070ff;
+$color-2: #172b4d;
+$color-3: #344563;
+$color-4: #7f8cff;
+
+/**
+*! Removing default browser style
+**/
+
+* {
+  box-sizing: border-box;
+
+  body {
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    max-width: 23.75rem;
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+  }
+
+  a {
+    text-decoration: none;
+  }
+
+  li {
+    list-style: none;
+  }
+}
+
+/**
+*! Header styles inputs here
+**/
+
+.header {
+  width: 100%;
+  display: flex;
+  align-items: center;
+}
+
+.navbar {
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.hamburger {
+  cursor: pointer;
+
+  [class^="bar"] {
+    display: block;
+    background-color: $color-1;
+    width: 1.875rem;
+    height: 0.1875rem;
+    margin: 0.3125rem;
+    margin-right: 2rem;
+  }
+}
+
+.logo {
+  font-weight: 700;
+  font-size: 1.125rem;
+  line-height: 1.25rem;
+  margin-left: 1rem;
+  color: $color-1;
+  display: flex;
+  align-items: center;
+}
+
+/**
+*! Main section styles inputs here
+**/
+
+.headline {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background-image: url("./images/header-shapes\ mobile.svg");
+  background-repeat: no-repeat;
+  background-position: top left;
+  background-color: #ffff;
+  height: 36.25rem;
+}
```

Comment:
- [ ] Dear @tsheporamantso, You did a great job adding the background image. But, as the previous reviewer mentioned a single background image is not compatible with different mobile screen sizes, kindly download each piece of the image alone from the Figma design and then try to set each piece to its own position in the different sections of your page.

Reply:
Dear @Tadesse-Alemayehu, I gladly appreciate your feedback, but I beg to differ and I would kindly refer you to the attached tutorial https://www.loom.com/embed/167236d17f104fc18298c5c9888354c9 regarding Figma and exporting files. Please check from 4 minutes, the presenter clearly states that it is impractical to export every single image, it's best you export a group of all shapes. I kindly ask for further elaboration on why is it incorrect to export background images in the manner I did.